packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman.vue

```diff
@@ -613,7 +597,45 @@ div.section-row.timespan > div > span {
 	color: var(--text-color-subdued);
 }
 
-.run-button {
-	margin-right: auto;
+/** Override default accordion styles */
+.accordion-component:deep(.p-accordion-header-link) {
+	background-color: var(--surface-100);
+}
+
+.accordion-component:deep(.p-accordion-content) {
+	background-color: var(--surface-100);
+	padding-left: var(--gap);
+	padding-right: var(--gap);
+}
+
+/* Override grid template so output expands when sidebar is closed */
+.overlay-container:deep(section.scale main) {
+	grid-template-columns: auto 1fr;
+}
+
+/* Make the slider light grey */
+:deep(.slider-content) {
+	background-color: var(--surface-100);
+	border-right: 1px solid var(--surface-border-light);
+}
+
+:deep(.slider-content header) {
+	background: transparent;
+}
+
+:deep(.slider-tab) {
+	background-color: var(--surface-100);
+	border-right: 1px solid var(--surface-border-light);
+}
+
+:deep(.slider-tab header) {
+	background: transparent;
+}
```

Comment:
Would be better if you can just apply a class called `secondary` or `input-config` to the `tera-slider` and `tera-slider` would have the rules to handle that. 

Since I see this pattern used quite in other areas, this can also be cleaned up in another PR.